### PR TITLE
raftstore: add consistency check log

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

File: src/raftstore/store/store.rs [hunk 1]
```diff
@@ -1863,6 +1864,7 @@ fn verify_and_store_hash(region_id: u64,
               expected_index);
     }
 
+    info!("[region {}] save hash of {} for consistency check later.", region_id, expected_index);
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: no need to record the expected hash here?

Comment (BusyJay) [inline src/raftstore/store/store.rs]: No, if they are different, L1845 will show them all. If they are the same, we still can infer it from the Verify Admin Command log.

Comment (siddontang) [inline src/raftstore/store/store.rs]: got it

**Comment by siddontang** [comment]
PTAL @javaforfun

**Pull request merged**